modules/upgrade/pages/migrate/kubernetes/strimzi.adoc

```diff
@@ -0,0 +1,351 @@
+
+- TLS enabled for the Kafka API, HTTP Proxy, and inter-broker communication (RPC).
+- Certificate management through glossterm:cert-manager[`cert-manager`] by default.
+
+NOTE: Both https://github.com/strimzi/strimzi-kafka-operator/issues/3088[Strimzi] and the Redpanda Operator currently do not support Kerberos for Kafka.
+
+See xref:manage:kubernetes/security/index.adoc[].
+
+=== Storage
+
+Redpanda performs best with direct-attached storage (NVMe or SSDs) to ensure high performance. To minimize latency, use of Kubernetes local PersistentVolumes.
+
+See xref:deploy:deployment-option/self-hosted/kubernetes/k-requirements.adoc[].
+
+=== JVM
+
+Redpanda does not require a JVM because it is written in C++.
+
+=== Zookeeper
+
+Redpanda does not require ZooKeeper. Instead, Redpanda brokers run the glossterm:Raft[] consensus algorithm for cluster coordination.
+
+=== Cruise control
+
+Redpanda manages partition distribution and load balancing, eliminating the need for external tools like cruise control. See xref:manage:cluster-maintenance/cluster-balancing.adoc[].
+
+=== Networking
+
+The Redpanda Helm chart has built-in support for the following networking options:
+
+* *NodePort* for direct access to services from outside the cluster using ports on each node.
+* *Load Balancer* for distributing external traffic to the cluster nodes, typically managed by cloud providers.
+
+NOTE: Ingress and route configurations are not available in the Redpanda Helm chart. To use these Services, you can disable the default NodePort Service and deploy your own custom Service.
+
+See xref:manage:kubernetes/networking/index.adoc[].
+
+=== Observability
+
+Redpanda exposes metrics to an HTTP endpoint by default, facilitating straightforward integration with monitoring tools such as Prometheus.
+
+NOTE: Strimzi includes built-in support for https://strimzi.io/docs/operators/latest/overview#metrics-overview-tracing_str[distributed tracing with OpenTelemetry]. However, the Redpanda Helm chart lacks support for distributed tracing.
+
+See xref:manage:kubernetes/monitoring/k-monitor-redpanda.adoc[] and xref:reference:public-metrics-reference.adoc[].
+
+=== Custom resources
+
+The deployment of Kafka components onto a Kubernetes cluster using both Strimzi and Redpanda Operator is configurable through custom resources. These resources are instances of APIs introduced by custom resource definitions (CRDs), which extend Kubernetes resources.
+
+CRDs act as configuration instructions to describe the custom resources in a Kubernetes cluster.
+CRDs also allow resources to benefit from native Kubernetes features like CLI accessibility and configuration validation.
+
+Both Strimzi and the Redpanda Operator offer CRDs for creating and managing components as Kubernetes resources. Some custom resources in Strimzi do not have an equivalent custom resource in Redpanda.
+
+[cols="3", options="header"]
+|===
+| Strimzi custom resource | Redpanda equivalent | Notes
```

Comment:
I'd argue that the Note column really doesn't have notes. Perhaps rename "For additional information" or something like that?

Reply:
Agreed. Thanks! Ths table went through a few iterations and I didn't reevaluate that column.